osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -2717,6 +2717,36 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/crescent/images/cre.svg"
       },
       "coingecko_id": "crescent-network"
+    },
+    {
+      "description": "The native token of Oraichain",
+      "denom_units": [
+        {
+          "denom": "ibc/161D7D62BAB3B9C39003334F1671208F43C06B643CC9EDBBE82B64793C857F1D",
+          "exponent": 0,
+          "aliases": [
+            "orai"
```

Comment:
as per chain registry, this denom with 0 exponent should be uorai. we can't have two denoms with the name exact name. which is correct?

Reply:
if it's uorai, or different at all, then the ibc/ hash denom should also be different

Reply:
as confusing as it is, the minimal denom of the Oraichain network is orai. It was a mistake when we launched the mainnet. Atm ít's not worth changing back to uorai, so we let it be like that. What should I do to update the json file?

Reply:
thanks for confirming that 'orai' is the base denom. In this case, the chain registry should also be updated, as it currently has 'uorai' as the base denom. In this case I'd recommend changing the ^6 unit denom to all uppercase 'ORAI' to avoid the problematic ambiguity, in which case both this file and the chain registry should be updated. I don't mind updating the chain reg myself as long as we are in agreement on this.

Reply:
I agree. Let me update the assetlist on the chain registry side as well